modules/apps/29-fee/keeper/escrow_test.go

```diff
@@ -129,7 +173,7 @@ func (suite *KeeperTestSuite) TestDistributeFee() {
 			},
 			func() {
 				// check if the refund acc has been refunded the timeoutFee & recvFee
-				expectedRefundAccBal := refundAccBal.Add(defaultTimeoutFee[0]).Add(defaultRecvFee[0]).Add(defaultTimeoutFee[0]).Add(defaultRecvFee[0])
+				expectedRefundAccBal := refundAccBal.Add(defaultRecvFee[0]).Add(defaultRecvFee[0])
```

Comment:
I guess the idea here is that the timeout fee was never escrowed?

Reply:
correct. This is because the hard-coded values are such that `recv (100) + ack (200) = timeout (300)`